application/views/collection_instrument_view.py

```diff
@@ -0,0 +1,44 @@
+from application.controllers.basic_auth import auth
+from application.controllers.collection_instrument import CollectionInstrument
+from flask import Blueprint
+from flask import make_response, request, jsonify
+from structlog import get_logger
+
+log = get_logger()
+
+collection_instrument_view = Blueprint('collection_instrument_view', __name__)
+
+
+@auth.login_required
+@collection_instrument_view.route('/upload/<exercise_id>/<ru_ref>', methods=['POST'])
+def upload_collection_instrument(exercise_id, ru_ref):
+    file = request.files['file']
+    code, msg = CollectionInstrument().upload_instrument(exercise_id, ru_ref, file)
+    return make_response(msg, code)
+
+
+@auth.login_required
+@collection_instrument_view.route('/download_csv/<exercise_id>', methods=['GET'])
+def download_csv(exercise_id):
+    code, msg = CollectionInstrument().get_instruments_by_exercise_id_csv(exercise_id)
+    response = make_response(msg, code)
+    log.info(code)
+    if code == 200:
+        response.headers["Content-Disposition"] = "attachment; filename=download.csv"
```

Comment:
One last thing. The final name being `download.csv` might be nicer as something more meaningful?